templates/react/docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+services:
```

Comment:
See that you also put compose files into the specific tech directory. It was something that I wasn't so sure about when I was making django template. I figured that it  would, be easier for some people to see what they need to copy into the compose file, but on the other hand, it might be a bit confusing, because in a real project the `react-compose` file and `django-compose` file are actually one file.

Reply:
yup, but i dont see any way around that, some initial setup will be required either way. 
Maybe something to consider would be mashing everything together but that does not seem right either.
At this point this solution seems most reasonable to me.